scripts/west_commands/runners/ezflashcli.py

```diff
@@ -39,13 +39,22 @@ def do_create(cls, cfg, args):
         return EzFlashCliBinaryRunner(cfg, tool=args.tool, sn=args.sn,
                                       erase=args.erase)
 
+    def get_load_offset(self):
+        if self.build_conf.getboolean('CONFIG_MCUBOOT'):
+            return 0
+        if self.build_conf.getboolean('CONFIG_HAS_FLASH_LOAD_OFFSET'):
+            return self.build_conf['CONFIG_FLASH_LOAD_OFFSET']
```

Comment:
Why is there a mcuboot exception?

Reply:
@andrzej-kaczmarek it seems you're the original author of this runner, but for others looking at this code, a comment describing what you wrote in above comment would be nice.